src/GitVersion.Core.Tests/Formatting/NumericFormatterTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using GitVersion.Formatting;
+
+namespace GitVersion.Tests.Formatting;
+
+[TestFixture]
+public class NumericFormatterTests
+{
+    [Test]
+    public void Priority_ShouldBe1() => new NumericFormatter().Priority.ShouldBe(1);
+    [Test]
```

Comment:
This is extremely nitty, but there should be a newline in between here. I wouldn't have pointed it out unless I had other questions about the PR. 

```suggestion
    public void Priority_ShouldBe1() => new NumericFormatter().Priority.ShouldBe(1);

    [Test]
```